indico/htdocs/sass/partials/_forms.scss

```diff
@@ -122,6 +122,10 @@ $i-form-button-line-height: 1.8;
             width: 400px;
         }
 
+        .form-field-warning {
```

Comment:
You can use `.i-form-field-fixed-width` instead, no need to declare a new class.